config/config.go

```diff
@@ -38,6 +38,9 @@ type Config struct {
 	// Databricks host (either of workspace endpoint or Accounts API endpoint)
 	Host string `name:"host" env:"DATABRICKS_HOST"`
 
+	// URL of the local metadata service that provides authentication credentials.
+	LocalMetadataServiceUrl string `name:"local_metadata_service_url" env:"DATABRICKS_LOCAL_METADATA_SERVICE_URL" auth:"local-metadata-service,sensitive"`
```

Comment:
Some of the other fields use initialisms, e.g. `AccountID`. To be consistent this should be `LocalMetadataServiceURL`.